lib/etre-client/client.rb

```diff
@@ -10,12 +10,14 @@ class Client
     API_ROOT = "/api/v1"
     META_LABEL_ID = "_id"
     META_LABEL_TYPE = "_type"
+    QUERY_TIMEOUT_HEADER = "X-Etre-Query-Timeout"
```

Comment:
I don't see where this is used, but rest looks ok.

Reply:
Ok, I'll remove that line. I ended up using a symbol instead of a string.